tests/index.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
What about looping over the configurations to make the test lighter?

```js
'use strict';

const { configs } = require('../lib');

describe('shareable configs', () => {
  Object.entries(configs).forEach(([configName, config]) => {
    it(`should export proper ${configName} config`, () => {
      expect(config).toMatchSnapshot()
    })
  })
});

```

Reply:
Even easier than that is to use [test.each](https://jestjs.io/docs/en/api#testeachtable-name-fn-timeout) so we have one definition with different values. I just add them like this to specify proper configs with snapshots, but I'm happy to implement these tests like that.

Reply:
Yes, why not!